viewing_party/party.py

```diff
@@ -1,23 +1,100 @@
-# ------------- WAVE 1 --------------------
+def create_movie(title,genre,rating):
+    if not bool(title) or not bool(genre) or not bool(rating):

```

Comment:
This works, but it's not necessary to cast to a bool to test.  `None` is falsey so it will evaluate to false.
```suggestion
    if not title or not genre or not rating:

```

Reply:
updated